packages/text-split/index.test.js

```diff
@@ -21,11 +21,14 @@ const propsBase = {
 }
 
 describe('<TextSplit />', () => {
-  it('should render a `.g-text-split` <div> root element', () => {
-    const { container } = render(<TextSplit {...propsBase} />)
+  it('should add a provided className to the root element', () => {
+    const className = 'my-nice-text-split'
+    const { container } = render(
+      <TextSplit {...propsBase} className={className} />
+    )
     const rootElem = container.firstChild
     expect(rootElem.tagName).toBe('DIV')
-    expect(rootElem).toHaveClass('g-text-split')
+    expect(rootElem).toHaveClass(className)
   })
```

Comment:
Just a meta observation across all of these PRs, I wonder if a test helper for this would be worthwhile:
```js
function assertAcceptsRootClassName(Component, className, props) {
  const { container } = render(<Component {...props} />)
  expect(container.firstChild).toHaveClass(className)
}

it ('should add a provided className to the root element', () => assertAcceptsRootClassName(TextSplit, className, props))
```

Reply:
Great point. Personally, somewhat on the fence about this - we used to have really heavily abstracted test code, and sometimes it felt like it made it much more difficult to understand and contribute to tests (unless you were already familiar with the abstractions). This feels super manageable though, so I might just be biased / haunted by that past experience 😅